From 05cd15be68cc0f60feca6bc949cc908135f2606a Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?=D0=A0=D1=83=D1=81=D0=BB=D0=B0=D0=BD=20=D0=98=D0=B6=D0=B1?=
 =?UTF-8?q?=D1=83=D0=BB=D0=B0=D1=82=D0=BE=D0=B2?= <lrn1986@gmail.com>
Date: Wed, 18 Mar 2015 14:06:33 +0000
Subject: [PATCH] Add more includes to shlobj.h

wtypesbase.h and wincrypt.h pull some other headers,
together they all provide EXTERN_C, DECLSPEC_IMPORT and WINAPI macros,
which are needed by shlobj.h directly or indirectly.
---
 mingw-w64-headers/include/shlobj.h | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/mingw-w64-headers/include/shlobj.h b/mingw-w64-headers/include/shlobj.h
index 3d2e21c..be087c4 100644
--- a/mingw-w64-headers/include/shlobj.h
+++ b/mingw-w64-headers/include/shlobj.h
@@ -6,6 +6,8 @@
 #ifndef _SHLOBJ_H_
 #define _SHLOBJ_H_
 
+#include <wtypesbase.h>
+#include <wincrypt.h>
 #include <winapifamily.h>
 
 #if WINAPI_FAMILY_PARTITION (WINAPI_PARTITION_DESKTOP) || defined(WINSTORECOMPAT)
-- 
1.8.5.3

